web/src/main/resources/application.properties-EXAMPLE

```diff
@@ -9,6 +9,7 @@ topbraid.password=
 
 # for crosswalk e.g. http://data.mskcc.org/ontologies/api/concept/crosswalk?vocabularyId={vocabularyId}&conceptId={conceptId}&histologyCode={histologyCode}&siteCode={siteCode}
 crosswalk.url=
+crosswalk.disable_cvs_querying=
```

Comment:
since the default in `CrosswalkRepository.java` is defined as `false`, I think that value should also be defined here in the example properties file with perhaps a comment indicating so

Reply:
change made and comment added